content/2.components/text-generate-effect.md

```diff
@@ -0,0 +1,22 @@
+---
+title: Text Generate Effect
+description: A cool text effect that fades in text on page load, one by one.
+---
+
+::ComponentLoader{label="Preview" componentName="TextGenerateDemo" type="examples"}  
+::
+
+## API
+
+| Prop Name     | Type     | Default  | Description                                               |
+| ------------- | -------- | -------- | --------------------------------------------------------- |
+| `words`        | `string` | Required | The text to be displayed with the generating effect.           |
+| `duration`    | `number` | `0.7`    | The duration of the text generation animation in seconds. |                           |
+| `filter`     | `boolean` | `true`   | The blur of the text.                                  |
+
+## Component Code
+
+You can copy and paste the following code to create these components:
+
+::CodeViewer{filename="TextGenerateEffect.vue" language="vue" componentName="TextGenerateEffect" type="ui" id="text-generate-effect"}
+::
```

Comment:
Hi @atif0075,
Please add credits to Aceternity UI component along with link.

Also add the component to Component Index (0.all.md)